elements/pl-order-blocks/dag_checker.py

```diff
@@ -45,27 +46,92 @@ def check_grouping(order: list[str], group_belonging: Mapping[str, Optional[int]
     return len(order)
 
 
-def grade_dag(order: list[str], depends_graph: Mapping[str, list[str]], group_belonging: Mapping[str, Optional[int]]) -> tuple[int, int]:
+def dag_to_nx(depends_graph: Mapping[str, list[str]]) -> nx.DiGraph:
+    """Convert input graph format from HTML into NetworkX object to utilize their algorithms."""
```

Comment:
Nit: we're really far removed from HTML at this point, I don't think we should mention it here.

Reply:
sure.